src/index.html

```diff
@@ -16,17 +16,31 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="page">
+    <div class="search__box">
+      <form
+        class="search-bar search-bar__big"
+        data-qa="big"
+      >
+        <input
+          class="search-bar__search search-bar__search--big"
+          type="text"
+          data-qa="keypress"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <form
+        class="search-bar search-bar__small"
+        data-qa="small"
+      >
+        <input
+          class="search-bar__search search-bar__search--small"
+          type="text"
+          data-qa="hover"
+          placeholder="Try “Los Angeles“"
+        />
+      </form>
+    </div>
   </body>
```

Comment:
You are not using semantic tags. Instead of using `div` for the `search__box`, consider using a `section` or `article` tag. Semantic tags improve your page's SEO and help screen readers.